app/views/api/backend_api_configs/edit.html.slim

```diff
@@ -0,0 +1,4 @@
+= content_for :sublayout_title, 'Edit Backend API'
```

Comment:
```suggestion
= content_for :sublayout_title, 'Edit Backend'
```

Reply:
mmm, you don't edit the backend. Not sure, maybe we should call them Backend Integrations but lets just stick with backends for now.